src/pages/saksbehandling/lukkSøknad/Avvist.tsx

```diff
@@ -0,0 +1,126 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { FormikErrors } from 'formik';
+import { Fareknapp, Knapp } from 'nav-frontend-knapper';
+import { RadioPanelGruppe, Textarea } from 'nav-frontend-skjema';
+import React, { SetStateAction, SyntheticEvent, useCallback } from 'react';
+
+import { JaNeiSpørsmål } from '~components/FormElements';
+import { hentLukketSøknadBrevutkast } from '~features/saksoversikt/sak.slice';
+import { useI18n } from '~lib/hooks';
+import * as Routes from '~lib/routes';
+import { useAppDispatch, useAppSelector } from '~redux/Store';
+import { LukkSøknadType } from '~types/Søknad';
+
+import nb from './lukkSøknad-nb';
+import styles from './lukkSøknad.module.less';
+import { AvvistBrevtyper, LukkSøknadFormData, setAvvistBrevConfigBody } from './lukkSøknadUtils';
+
+const Avvist = (props: {
+    values: LukkSøknadFormData;
+    errors: FormikErrors<LukkSøknadFormData>;
+    setValues: (values: SetStateAction<LukkSøknadFormData>, shouldValidate?: boolean | undefined) => unknown;
+    handleChange: (event: React.ChangeEvent<HTMLTextAreaElement> | SyntheticEvent<EventTarget, Event>) => void;
```

Comment:
Samme som jeg skrev for `Trukket`; går det an å begrense ansvarsområdet litt for denne komponenten?